docs/ref/contrib/admin/javascript.txt

```diff
@@ -29,7 +29,13 @@ In your custom ``change_form.html`` template, extend the
 
     {% block admin_change_form_document_ready %}
     {{ block.super }}
-    <script type="text/javascript">
+    <script type="text/javascript" src="{% static 'app/formset_handlers.js' %}></script>
+    </script>
+    {% endblock %}
+
+.. code-block:: javascript
```

Comment:
You can use the `.. snippet::` directive to render a nice block with the filename.